.github/workflows/build.yml

```diff
@@ -31,3 +31,21 @@ jobs:
           ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
         run: |
           ./build.sh --include-user-channel --token "$ANACONDA_TOKEN" --push
+
+  build-macos-conda-packages:
```

Comment:
Does this run concurrently with the Linux build....in parallel. We want to make sure they are not sequential as it will take forever.

Reply:
Yes - running in parallel as I write this.  The builds are **not** sequential, as I laid it out in the workflow.